app/controllers/search_controller.rb

```diff
@@ -1,5 +1,13 @@
 class SearchController < ApplicationController
   def new
-    
+
+  end
+
+  def create
+    session[:range] = params[:ev_range]
+    session[:activity] = params[:activity]
+    session[:zipcode] = params[:zipcode]
+
```

Comment:
Using individual keys for session works for now but I would say it would be best to have one object stored in the session `session[:search] = { activity: params[:activity] }`